### PR TITLE
blog render.com

### PR DESCRIPTION
**Pull request by alex-fusionauth**

### CODE REVIEW

File: astro/src/content/blog/fusionauth-on-render.mdx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+### Deploy The Web Service
+
+Click the `Deploy Web Service` button. Wait for the deployment process to complete. You’ll see a loading screen as the service starts.
+
+![Logs showing in Render](/img/blogs/fusionauth-on-render/6-fusionauth-render.jpg)
+
+Once the logs show `HTTP server listening on port [9011]` you will be ready to access the web service.
+
+### Enter Maintenance Mode
```
Comment (mark-robustelli) [inline astro/src/content/blog/fusionauth-on-render.mdx:111]: Would it be easier to use a kickstart here?

File: astro/src/content/blog/fusionauth-on-render.mdx [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  })
+);
+
+const server = app.listen(port, () =>
+  console.log(`Proxy server listening on port ${port}!`)
+);
+server.keepAliveTimeout = 120 * 1000;
+server.headersTimeout = 120 * 1000;
+```
+3. Deploy this application to Render under a new web service named `FusionAuth Render Proxy`.
```
Comment (mark-robustelli) [inline astro/src/content/blog/fusionauth-on-render.mdx:181]: Should there be steps here to help show how to deploy?

**Review by mark-robustelli — CHANGES_REQUESTED** [review]
Some comments and a few changes.  I think you mentioned the images will be re-done but they are very hard to read on my machine.

File: astro/src/content/blog/fusionauth-on-render.mdx [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+authors: Alex Patterson
+categories: Tutorial
+image: /img/blogs/fusionauth-on-fly-io/running-fusionauth-on-fly.io-2.png
+tags: render, self-hosted
+excerpt_separator: "{/* more */}"
+---
+import Aside from '../../components/Aside.astro';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+In this tutorial, you will install FusionAuth on [Render](https://render.com/) using Postgres and a Node.js proxy.
```
[the remark this answers is not in the document]
Comment (alex-fusionauth) [reply astro/src/content/blog/fusionauth-on-render.mdx:14]: It is a blog that is in the category of Tutorial, like this...
https://fusionauth.io/blog/fusionauth-on-fly-io

File: astro/src/content/blog/fusionauth-on-render.mdx [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+First things first, you need to set up our PostgreSQL database.
+
+### Create A New PostgreSQL Database
+
+1. Go to your Render dashboard and create a new PostgreSQL instance.
+2. Name this instance `fusionauth-example`.
+    
+    ![PostgreSQL](/img/blogs/fusionauth-on-render/1-fusionauth-render.jpg)
+3. Choose `Ohio (US East)` as the region.
```
[the remark this answers is not in the document]
Comment (alex-fusionauth) [reply astro/src/content/blog/fusionauth-on-render.mdx:48]: I don't want confusion around the guide vs. the video.

File: astro/src/content/blog/fusionauth-on-render.mdx [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+Next, you need to set up FusionAuth as a web service.
+
+1. In your Render dashboard, create a new web service.
+2. Use an existing image, hosted on docker.
+3. Specify the image: `docker.io/fusionauth/fusionauth-app:latest`.
+    
+```plaintext
+docker.io/fusionauth/fusionauth-app:latest
+```
+4. Choose `Ohio (US East)` as the region again.
```
[the remark this answers is not in the document]
Comment (alex-fusionauth) [reply astro/src/content/blog/fusionauth-on-render.mdx:69]: yep region
image

File: astro/src/content/blog/fusionauth-on-render.mdx (hunk 1 above)
Comment (alex-fusionauth) [reply astro/src/content/blog/fusionauth-on-render.mdx:111]: That is not possible. This has to be a docker image.

File: astro/src/content/blog/fusionauth-on-render.mdx (hunk 2 above)
Comment (alex-fusionauth) [reply astro/src/content/blog/fusionauth-on-render.mdx:181]: It is so straight forward and covered in the video. I really want to push them to the video this is just an incase they need to reference.

**Review by mark-robustelli — DISMISSED** [review]

**Review by mark-robustelli — APPROVED** [review]

**Pull request merged**